dbstore.go

```diff
@@ -369,8 +381,19 @@ func (p *persister) save(ws *Worksheet) error {
 			Value:       p.writeValue(value),
 		})
 
+		if _, ok := value.(*Slice); !ok {
```

Comment:
same here -- this would be clearer as an if/else

Reply:
yes, changed as suggested